src/app/features/switch-account-drawer/components/create-account-action.tsx

```diff
@@ -7,8 +7,16 @@ interface CreateAccountActionProps {
 }
 export function CreateAccountAction({ onCreateAccount }: CreateAccountActionProps) {
   return (
-    <Flex py="space.05" px="space.05" flexGrow="1">
-      <LeatherButton fullWidth onClick={() => onCreateAccount()}>
+    <Flex
+      mt="100px"
+      py="space.05"
+      px="space.05"
+      flexGrow="1"
+      position="fixed"
+      bottom={0}
+      width="100%"
+    >
+      <LeatherButton data-testId="create-account" fullWidth onClick={() => onCreateAccount()}>
```

Comment:
Shouldn't this be `data-testid`?

Reply:
Thanks, I'll remove this as it's not used in tests, it was just helping me search the DOM